community/detectors/apache_druid_preauth_rce_cve_2021_25646/src/test/java/com/google/tsunami/plugins/detectors/rce/cve202125646/ApacheDruidPreAuthRCECVE202125646VulnDetectorTest.java

```diff
@@ -0,0 +1,123 @@
+package com.google.tsunami.plugins.detectors.rce.cve202125646;
```

Comment:
Same here, please add the license comment.

Reply:
done

> Same here, please add the license comment.

done.